src/data/Storage.ts

```diff
@@ -93,6 +93,10 @@ function _todayPeriod(countOfCycles: number): Promise<void> {
     });
   }
 
+  if (countOfCycles > 1) {
+    cycles[0].cycleLength = 0;
```

Comment:
Why?

Reply:
Because the first cycle is the current one.  Its length is filled in only after clicking the `Mark button` .